ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -15,13 +15,14 @@
 open Stdext
 open Listext
 open Threadext
+open Xstringext
 
 module D = Debug.Make(struct let name = "xapi_pvs_proxy_control" end)
 open D
 
 let proxy_port_name vif =
-  let uuid = String.sub vif.API.vIF_uuid 0 8 in
-  Printf.sprintf "pvs-%s" uuid
+  let mac = String.replace ":" "" vif.API.vIF_MAC in
```

Comment:
Cool. Maybe worth a comment explaining the maximum string length that you found for interface names, to explain the `String.replace`?